docs/source/nn.rst

```diff
@@ -4,6 +4,15 @@
 torch.nn
 ===================================
 
+.. note:: The contents of this page are no longer maintained. Click here to see the `stable version of torch.nn <https://pytorch.org/docs/stable/nn/index.html>`_.
```

Comment:
Instead of having nn/index.rst, couldn't we have put the contents of that file into this one (nn.rst)? This would avoid needing to have a deprecation